.github/workflows/run-short-tests.yaml

```
@@ -1,16 +1,21 @@
-name: Run tests
+name: Run Short Tests
 
-on: [push]
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+  workflow_dispatch:
```

```suggestion
on: [pull_request, workflow_dispatch]
```
I think we are fine with default types of pull_request (which are `opened`, `synchronize`, `reopened`)

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

---

Pull request types include `edited`, `labeled`, `assigned`, `review_requested`, and others that would needlessly start a build. I think. So I prefer to leave it as is.

---

I did not see that! Taking the suggestion!